### ISSUE TITLE
Scroll on mobile

### ISSUE THREAD

**Issue by ortigozamatias** [rsuite/rsuite-table#155]
Hi,

I've started a brand new project with this library and `create-react-app` and looks good on bigger devices yet on mobile it's really hard to scroll. It takes several attempts to scroll up or down.

Here it is the code I'm using:

```javascript
    <Table
      autoHeight
      bordered
      data={users}
      virtualized
    >
      <Table.Column flexGrow={1}>
        <Table.HeaderCell>Id</Table.HeaderCell>
        <Table.Cell dataKey="id" />
      </Table.Column>
    </Table>
```

I've tried to wrap it inside a `div` with `overflow: auto` but it doesn't seem to be helping.

Thanks in advance!

**Comment by simonguo** [comment]
Using tables on mobile is not a good design, and it is not recommended to use rsuite-table on mobile

**Comment by ortigozamatias** [comment]
Not even if it has a single column?

**Comment by ortigozamatias** [comment]
It seems to be appearing in touch devices. I've tried it on an iPad Pro and the scroll is not working either. In this case table has 6 columns.
When running in DevTools, this error appears in the console:

`[Intervention] Ignored attempt to cancel a touchmove event with cancelable=false, for example because scrolling is in progress and cannot be interrupted.`

**Comment by dkoloditch** [comment]
+1. Vertical (touch) scrolling on mobile is really "sticky". I've had multiple complaints about this experience.

**Comment by dkoloditch** [comment]
@ortigozamatias Did you ever find a solution to this issue? Based on a quick examination, I'm guessing the CSS scrolling functionality is being overridden in favor of handling at the JS level. I've tried a number of styling overrides with no luck.

In regard to this component and tables being incompatible with mobile best practices, this seems problematic for a few reasons:
1. There's no warning in the docs to avoid using this component for mobile.
2. With this exception, this component is quite mobile friendly, which gives a false impression of support if mobile table usage is officially discouraged.
3. Since an effort has clearly already been made to make this component mobile friendly, consistency would suggest continuing efforts in this regard and leaving best practices to user discretion.
4. There are no alternatives in the RSuite ecosystem with the level of functionality (sorting, pagination, etc) this component provides that can quickly and easily take its place.

**Comment by simonguo** [comment]
hi all
There are two operations for scrolling:
1. Drag the scroll bar to scroll
2. Scroll through the scroll wheel in the table body

Which one is your failure?

**Comment by dkoloditch** [comment]
Hi @simonguo. The issue is with touch scrolling on mobile. See the `Virtualized` table example [on the RSuite site](https://rsuitejs.com/components/table/#virtualized). Notice on desktop the scrolling 'coasts'. That is, you when you initiate scrolling on a large table, it will coast to a stop. Now view the same table using the browser's mobile tool, a phone, or a tablet. Notice the touch scrolling is 'sticky' and you cannot easily scroll through a large table of data. The scrolling comes to an immediate stop as soon as you lift your finger. This is the failure.

**Comment by simonguo** [comment]
hi @dkoloditch  fixed in #263

**Comment by dkoloditch** [comment]
> hi @dkoloditch fixed in #263

@simonguo 🙌 🙌 🙌 this looks great, thanks!

**Comment by simonguo** [comment]
@dkoloditch  In fact, there are still some browser compatibility issues, such as a white screen when scrolling on iOS Chrome. Currently still trying to fix it. If you have any solutions, you can tell us.
